geniza/corpus/management/commands/export_metadata.py

```diff
@@ -1,9 +1,106 @@
+import os
 import time
 
+from django.conf import settings
 from django.core.management.base import BaseCommand
 from django.utils import timezone
+from git import Repo
 
-from geniza.corpus.metadata_export import DocumentExporter
+from geniza.corpus.metadata_export import PublicDocumentExporter
+
+
+class MetadataExportRepo:
+    local_path_settings_key = "METADATA_REPO_LOCAL_PATH"
+    remote_url_settings_key = "METADATA_REPO_REMOTE_URL"
+    default_remote_url_org = "https://github.com/Princeton-CDH"
+
+    repo_name = "test-geniza-metadata"
+    repo_dir_data = "data"
+    ext_csv = ".csv"
+
+    def __init__(self, local_path=None, remote_url=None):
+        self._local_path = local_path
+        self._remote_url = remote_url
+        self._repo = None
+
+    @property
+    def local_path(self):
+        if not self._local_path:
+            if hasattr(settings, self.local_path_settings_key):
+                self._local_path = getattr(settings, self.local_path_settings_key)
+            else:
+                self._local_path = os.path.abspath(
+                    os.path.join(settings.BASE_DIR, "..", self.repo_name)
+                )
+        return self._local_path
```

Comment:
my preference would be to require the configuration in local settings and raise a `CommandError` with an appropriate message if it isn't present (at least, for any setting we want to require and make sure is configured properly)